lib/board.rb

```diff
@@ -0,0 +1,21 @@
+class Board
+  def initialize(cells)
+    @cells = cells
+  end
+
+  def get_board()
+    cells.map { |cell| cell.symbol }
```

Comment:
@ssunday mentioned this on the initial review

We can do this instead.
```suggestion
    cells.map(&:symbol)
```